README.md

```diff
@@ -42,8 +42,9 @@ module "db" {
   backup_window      = "03:00-06:00"
 
   # Enhanced Monitoring - see example for details on how to create the role
+  # by yourself, in case you don't want to create it automatically
   monitoring_interval = "30"
-  monitoring_role_arn = "arn:aws:iam::123456789012:role/rds-monitoring-role"
+  monitoring_role_autocreate = "rds-monitoring-role"
```

Comment:
`monitoring_role_autocreate` sounds like a boolean, not string.
Let's call this as `monitoring_role_name`, and use `create_monitoring_role` as the main switch - true/false.

Reply:
So I should additional variable `create_monitoring_role` for the switch and set some default name for `monitoring_role_name`?

Reply:
yes, and the defaults are:
```
create_monitoring_role = false
monitoring_role_name = ""
```